phpBB/phpbb/messenger/method/phpbb_email.php

```diff
@@ -0,0 +1,637 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\messenger\method;
+
+use Symfony\Component\Mailer\Transport;
+use Symfony\Component\Mailer\Mailer;
+use Symfony\Component\Mime\Address;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\Mime\Header\DateHeader;
+use Symfony\Component\Mime\Header\Headers;
+use Symfony\Component\Mime\Header\IdentificationHeader;
+use Symfony\Component\Mime\Header\MailboxHeader;
+use Symfony\Component\Mime\Header\MailboxListHeader;
+use Symfony\Component\Mime\Header\PathHeader;
+use Symfony\Component\Mime\Header\UnstructuredHeader;
+
+/**
+ * Messenger class
+ */
+class phpbb_email extends base
+{
+	/** @var array */
+	private const PRIORITY_MAP = [
```

Comment:
Why are we duplicating this from symfony?

Reply:
It turned out it's just to set the custom `X-MSMail-Priority` which doesn't present in Symfony mailer.